package.json

```diff
@@ -307,7 +307,7 @@
         "@box/blueprint-web": "^7.36.3",
         "@box/blueprint-web-assets": "^4.28.0",
```

Comment:
does blueprint-web-assets peer dependency need to be updated?

Reply:
yes it should, thank for pointing that up!